src/Nethermind/Nethermind.Core/Account.cs

```diff
@@ -78,7 +78,7 @@ private Account(Account account, in UInt256 nonce, in UInt256 balance)
         public bool HasStorage => _storageRoot is not null;
 
         public UInt256 Nonce { get; }
-        public UInt256 Balance { get; }
+        public readonly UInt256 Balance;
```

Comment:
Why this pattern?
Why not this pattern for everything else?

Reply:
Only changed the ones I needed a `ref ` to

Reply:
Changed back to property, but is more changes

Reply:
Yes I've seen it later. This was only a question :)
 IMO it is fine to be readonly field, but maybe we need more readonly fields ? :)

Reply:
Change back to field and also the other one